File: PRAK304/PRAK304-2310817110011-Ghani_Mudzakir.py
nilai= int(input())

if (nilai == 0):

    print("Nol")

elif(nilai > 0 and nilai < 10):

    print("Satuan")

elif(nilai > 10 and nilai <= 19):

    print("Belasan")

elif (nilai > 99):

    print("Anda Menginput Melebihi Limit Bilangan")

else:
    print("Puluhan")

File: PRAK304/PRAK304-2310817110011-Ghani_Mudzakir.c
#include <stdio.h>

int main()
{
    int nilai;
    scanf("%d", &nilai);

    if (nilai == 0)
    {
        printf("Nol");
    }
    else if(nilai > 0 && nilai < 10)
    {
        printf("Satuan");
    }
    else if(nilai > 10 && nilai <= 19)
    {
        printf("Belasan");
    }
    else if (nilai > 99)
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    else{
        printf("Puluhan");
    }

    
    return 0;
}




